mlrun/db/nopdb.py

```diff
@@ -23,6 +23,7 @@
 import mlrun.errors
 
 from ..config import config
+from ..lists import RunList
```

Comment:
relative imports are discouraged as they cause circular import errors. most of them are due to legacy reasons but we decided as a convention to use full imports.